### PR TITLE
[Cleanup]  Adjusting Column Width For Notes

### PR DESCRIPTION
**Pull request by Civolilah**
@beganovich @turbo124 The PR includes the notes column's width adjustment. Screenshot:

Screenshot 2024-05-20 at 19 22 08

Let me know your thoughts.

### CODE REVIEW

**Comment by turbo124** [comment]
@beganovich ping

File: src/common/hooks/useExtractTextFromHTML.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+export function useExtractTextFromHTML() {
```
Comment (beganovich) [inline src/common/hooks/useExtractTextFromHTML.ts:11]: Is there a reason why this is marked as a hook?

Seems like more work and more lines of code if this is marked as hook, but it has no access to React's lifecycle nor state.

File: src/common/hooks/useSanitizeHTML.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import DOMPurify from 'dompurify';
+
+export function useSanitizeHTML() {
```
Comment (beganovich) [inline src/common/hooks/useSanitizeHTML.ts:13]: Same as previous one.

**Review by beganovich — COMMENTED** [review]
Looks good, please just clarify the situation about hooks.

**Comment by Civolilah** [comment]
> Looks good, please just clarify the situation about hooks.

@beganovich Yes, good catch. It can be converted to a clear function instead of hooks since we do not use any states in the logic. It has been converted, please let me know your thoughts.

**Pull request merged**